recipes/sample-sheet/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "sample-sheet" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: "{{ name|lower }}"
```

Comment:
Also not critical: the name does not need to be lower-cased - it already is lower case.